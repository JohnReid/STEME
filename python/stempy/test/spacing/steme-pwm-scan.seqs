44, Test-spacing-seq-1
44, Test-spacing-seq-2
44, Test-spacing-seq-3
