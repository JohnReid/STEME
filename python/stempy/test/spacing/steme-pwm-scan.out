MOTIF-1,AAGGTTCC,0,12,+,0.981141,800,1.52587890625e-05
MOTIF-1,AAGGTTCC,1,12,+,0.981141,800,1.52587890625e-05
MOTIF-1,AAGGTTCC,2,12,-,0.981141,800,1.52587890625e-05
MOTIF-2,CCTTGGAA,0,24,+,0.981141,800,1.52587890625e-05
MOTIF-2,CCTTGGAA,1,24,+,0.981141,800,1.52587890625e-05
MOTIF-2,CCTTGGAA,2,24,+,0.981141,800,1.52587890625e-05
